packaging/windows/product.wxs.template

```diff
@@ -95,12 +101,24 @@
         <CustomAction Id="RemoveCrcDaemonTask" BinaryKey="WixCA" DllEntry="WixQuietExec" Execute="deferred" Impersonate="no" Return="ignore" />
         <SetProperty Action="CAInstallHyperv" Id="InstallHyperv" Value='"[System64Folder]dism.exe" /online /enable-feature /featureName:microsoft-hyper-v-all /NoRestart /quiet' Before="InstallHyperv" Sequence="execute"/>
         <CustomAction Id="InstallHyperv" BinaryKey="WixCA" DllEntry="WixQuietExec64" Execute="deferred" Impersonate="no" Return="ignore" />
-        <SetProperty Action="CAAddUserToHypervAdminGroup" 
-            Id="AddUserToHypervAdminGroup" 
-            Value="&quot;[POWERSHELLEXE]&quot; -NonInteractive -ExecutionPolicy Bypass -NoProfile -Command &quot;Add-LocalGroupMember -Member [LogonUser] -SID S-1-5-32-578&quot;" 
-            Before="AddUserToHypervAdminGroup" 
+        <SetProperty Action="CAAddUserToHypervAdminGroup"
+            Id="AddUserToHypervAdminGroup"
+            Value="&quot;[POWERSHELLEXE]&quot; -NonInteractive -ExecutionPolicy Bypass -NoProfile -Command &quot;Add-LocalGroupMember -Member [LogonUser] -SID S-1-5-32-578&quot;"
+            Before="AddUserToHypervAdminGroup"
             Sequence="execute"/>
         <CustomAction Id="AddUserToHypervAdminGroup" BinaryKey="WixCA" DllEntry="WixQuietExec64" Execute="deferred" Impersonate="no" Return="ignore" />
+        <SetProperty Action="CACreateSMBShare"

```

Comment:
Just wondering; can this be optional? As in certain environment the use of workstations running print/filesharing is not allowed. Not sure how the installer handles this or how admins would appreciate this. The problem might be that the tooling side expects this; I do not remember seeing a check for this on startup.

We might have to create a follow-up, as when no setup/installer is delivered this becomes a manual step or something that needs to be checked for.

Reply:
this custom actions return value is ignored so even if this fails the installation will succeed to take care of situations where this is disabled using group or domain policies or any other errors, and during start when mounting the shared directory there's a check to see if the 'crc-dir0' share exists before attempting to mount

we need to have a generic todo to remove all the privileged setup operations from the msi and bring them back to `crc setup` again, i think this'll be best handled with that task once we are finally removing the msi